<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>udpClient</class>
 <widget class="QDialog" name="udpClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>646</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>udpClient</string>
  </property>
  <widget class="QListWidget" name="m_context">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>391</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>380</y>
     <width>361</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="m_port">
      <property name="text">
       <string>端口</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="port"/>
    </item>
    <item>
     <widget class="QPushButton" name="m_start">
      <property name="text">
       <string>Begin Receive</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
